include/net/net_pkt.h

```diff
@@ -419,12 +419,17 @@ static inline void net_pkt_set_priority(struct net_pkt *pkt,
 {
 	pkt->priority = priority;
 }
-#else
+#else /* NET_TC_COUNT == 1 */
+#define net_pkt_set_priority(...)
+
 static inline u8_t net_pkt_priority(struct net_pkt *pkt)
 {
 	return 0;
 }
-#endif
+
+#define net_pkt_set_priority(...)
```

Comment:
Why repeated twice? Why not use real params instead of "..."?

Reply:
Both ways can be used here. The benefit of ... is that if the number of parameters change in that func, we do not need to change the macro.

Reply:
> Why repeated twice?

That is actually correct, the net_pkt_set_priority() is set twice here.

Reply:
> The benefit of ... 

Ok, if it's common pattern, sounds good, somehow I didn't pay attention to that.

Reply:
> That is actually correct, the net_pkt_set_priority() is set twice here.

Why? Why have stuff like:

~~~
#define net_pkt_set_priority(...)
// foo
#define net_pkt_set_priority(...)
~~~

Reply:
> That is actually correct, the net_pkt_set_priority() is set twice here.
> 
> Why? Why have stuff like:

I meant your comment is correct, not the actual code :)